Misc/NEWS.d/next/Tools-Demos/2025-08-01-13-27-43.gh-issue-137025.ubuhQC.rst

```diff
@@ -0,0 +1,3 @@
+The ``wasm_build.py`` script has been removed.  ``Tools/wasm/emscripten``
```

Comment:
Should we actually remove it? I think it still exists and is just broken.

Reply:
I think it's already gone, as of ee49644cc9975f0ad3756b394b77043195ea08fc .

`find . -name wasm_build.py` in a fresh clone gives no results.